c2cgeoportal/scaffolds/update/CONST_vars.yaml_tmpl

```diff
@@ -245,6 +245,10 @@ vars:
     # architecture.
     external_themes_url:
 
+    # Available options for http sessions created within c2cgeoportal
+    http_options:
```

Comment:
Should be added in CONST_config-schema.yaml

Reply:
OK